gamemodes/terrortown/gamemode/server/sv_player.lua

```diff
@@ -1477,3 +1481,15 @@ end
 function GM:PlayerShouldTaunt(ply, act)
 	return false
 end
+
+---
+-- Called by @{GM:EntityTakeDamage} after damage output was changed due to internal events. You can modify all parts of the damage info in this hook again.
+-- @param Entity ent The @{Entity} taking damage
+-- @param CTakeDamageInfo dmginfo Damage info
+-- @return boolean Return true to completely block the damage event
+-- @note e.g. no damage if a value after Karma calculation is too low, etc
+-- @hook
+-- @realm server
+-- @local
+function GM:TTT2PostEntityTakeDamage(ent, dmginfo)
+end
```

Comment:
```suggestion
-- Called by @{GM:EntityTakeDamage} after damage output was changed due to internal events. You can modify all parts of the damage info in this hook again.
-- @param Entity ent The @{Entity} taking damage
-- @param DamageInfo dmgInfo The damage info
-- @return boolean Return true to completely block the damage event
-- @note There is no damage value if it is too low after the karma calculation.
-- @hook
-- @realm server
-- @local
function GM:TTT2PostEntityTakeDamage(ent, dmgInfo)

end
```

Reply:
The note was meant as an example what you could do with the post damage hook. But I removed that note instead.